nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -219,6 +219,7 @@ NIX_PATH="nixpkgs=/tmp/root/$nixpkgs:nixos-config=$NIXOS_CONFIG" NIXOS_CONFIG= \
 # Copy the NixOS/Nixpkgs sources to the target as the initial contents
 # of the NixOS channel.
 mkdir -m 0755 -p $mountPoint/nix/var/nix/profiles
+mkdir -m 1777 -p $mountPoint/nix/var/nix/profiles/nixup
```

Comment:
Do we actually need a new subdirectory for nixup profiles? For instance, per-user channels use e.g. `/nix/var/nix/profiles/per-user/root/channels`